appium/webdriver/extensions/execute_driver.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+
+from ..mobilecommand import MobileCommand as Command
+
+
+class ExecuteDriver(webdriver.Remote):
+
+    def execute_driver(self, script, script_type='webdriverio', timeout=None):
+        """Run a set of script against the current session, allowing execution of many commands in one Appium request.
+        Please read http://appium.io/docs/en/commands/session/execute-driver for more details about the acceptable
+        scripts and the output format.
+
+        Args:
+            script (string): The string consisting of the script itself
+            script_type (string): The name of the script type. Defaults to 'webdriverio'.
+            timeout (optional): The number of `ms` Appium should wait for the script to finish before killing it due to timeout.
+
+        Usage:
+            self.driver.execute_driver(script='return [];')
+            self.driver.execute_driver(script='return [];', script_type='webdriverio')
+            self.driver.execute_driver(script='return [];', script_type='webdriverio', timeout=10000)
+
+        Return:
```

Comment:
Return -> Returns

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings